### PR TITLE
fix(curriculum): explain the box-sizing property 

### LINKED ISSUE #49156: learn-the-css-box-model-by-building-a-rothko-painting challenge - missing information about different css box models
### Describe the Issue

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-the-css-box-model-by-building-a-rothko-painting/

Challenge called "Learn the **css box model** by..."
There are bunch of steps which describes some parts of the box model, a lot of steps about changing colors etc
But it looks like there is no step describing that there are different models available like content-box, border-box etc
And in the next project https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-css-flexbox-by-building-a-photo-gallery/step-7 there is a phrase:

> Normalize your box model by creating a * selector and setting the box-sizing property to border-box as the value.

But there was no information before that about border-box and what is it and why we are setting it to border-box here

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-the-css-box-model-by-building-a-rothko-painting/

### Steps to Reproduce

Just go through challenge

### Recommended fix or suggestions

It will great if "Learn the css box model" project will include information about different box-sizing values 

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by CBID2**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #49156 

I explained the `box-sizing property` in two separate steps. One focuses on the `content-box` value and its purpose while the other focuses on the `border-box` value and how it differs from the former. Through these changes, I anticipate that campers would gain a better understanding of the steps.

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6494da0daf5df5197963671d.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6494da0daf5df5197963671d.md:1]: This step is empty?

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6493bc0d99879635209565aa.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/6493bc0d99879635209565aa.md:1]: This step is empty?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
It looks like there is still some confusion here. Here are the steps to resolve the issue.

From a clean branch:

- Use the `insert-step` tool to insert a new step 7
  - The new step 7 should instruct the campers to set the `box-sizing` property to `content-box` and explain the behaviour and the browser default.
- Update the old step7, which will now be step 8
  - Step 8 should be changed to tell the campers to change `box-sizing` to `border-box` and explain why the change matters.

**Comment by CBID2** [comment]
> It looks like there is still some confusion here. Here are the steps to resolve the issue.
> 
> 
> 
> From a clean branch:
> 
> 
> 
> - Use the `insert-step` tool to insert a new step 7
> 
>   - The new step 7 should instruct the campers to set the `box-sizing` property to `content-box` and explain the behaviour and the browser default.
> 
> - Update the old step7, which will now be step 8
> 
>   - Step 8 should be changed to tell the campers to change `box-sizing` to `border-box` and explain why the change matters.

I'll try

**Comment by naomi-lgbt** [comment]
Closing in favour of #50795

**Pull request closed without merging**